scripts/linters/custom_eslint_checks/rules/no-multiline-disable.spec.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for the no-multiline-disable.js file.
+ */
+
+'use strict';
+
+var rule = require('./no-multiline-disable');
+var RuleTester = require('eslint').RuleTester;
+
+var ruleTester = new RuleTester();
+ruleTester.run('no-multiline-disable', rule, {
+  valid: [
+    `
+    // eslint-disable-next-line max-len
+    it('should' +
+    'happen')`,
+    `
+    /* eslint-disable-next-line max-len */
+    it('should' +
+    'happen')`,
+    `
+    /* eslint-disable no-multiline-disable */
```

Comment:
are we expecting `oppia/no-multiline-disable` here? OR it's oppia-linters context-specific test?

Reply:
I depend on what we are using as a rule name on line 25. If we use `oppia/no-multiline-disable` there, then we have to use it in the tests below. Else we can just use `no-multiline-disable`